README.md

```diff
@@ -1,15 +1,17 @@
 # Recipe API Consumer
 
+## Purpose
+Create a web application that will display recipes based on a search term. We will utilize an API from Edamam called the [Recipe Search API](https://developer.edamam.com/edamam-recipe-api).
+
+This is an individual, [stage 2](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md) project.
```

Comment:
Should we add a Trello requirement since we are not providing the project in waves to practice these skills?